sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Utility/ReporterUtils.cs

```diff
@@ -166,7 +166,7 @@ internal bool IsTimeGreaterThanCurrentPlus10Minutes(string sasUri)
                 Uri url = new Uri(sasUri);
                 string query = url.Query;
                 var queryParams = System.Web.HttpUtility.ParseQueryString(query);
-                string expiryTime = queryParams["se"]; // 'se' is the query parameter for the expiry time
+                string? expiryTime = queryParams["se"]; // 'se' is the query parameter for the expiry time
```

Comment:
Explicit nullable opt-in.   _([src](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger.csproj#L11))_